spring-webflux-threads/src/main/resources/application.yml

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
Can we delete this empty file or it's necessary?

Reply:
Yes, we can, this is a default boot properties class.